src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -31,10 +31,8 @@ export default function FormNav(props) {
   const pageList = createPageList(formConfig, formPages);
 
   const eligiblePageList = getActiveExpandedPages(pageList, formData);
-
-  const chapters = uniq(
-    eligiblePageList.map(p => p.chapterKey).filter(key => !!key),
-  );
+  const test = eligiblePageList.map(p => p.chapterKey);
```

Comment:
This is platform code if this code really does need to be modified needs to be discussed with VSP.

Reply:
Okay I took that out.